### DISCUSSION TITLE
Enterprise Edition 2.6.1 - Possible issue: multitrack-conference.html

### DISCUSSION THREAD

**Discussion by AppThisThing** [ant-media/Ant-Media-Server#5192] [Q&A] [answered]
**Issue:** Not passing arguments through url parameters

**ie.** multitrack-conference.html?streamId=myTest101

<input type="text" class="form-control" value="room1" id="roomName" placeholder="Type room name">

  /**
    * This page accepts 3 arguments through url parameter
    * 1. "streamId": the stream id to publish stream. It's optional. ?streamId=stream1
    * 2. "playOnly": If it's true, user does not publish stream. It only play streams in the room.
    * 3. "token": It's experimental.
    */

**Comment by mekya** [comment, marked as answer]
Hi @AppThisThing,

Thank you for asking this issue. 

I've just tested on my end and it's working as expected for me. Let me explain how it works

When you give the streamId parameter, it uses as your publishStreamId. I mean, after you click the join button, you should see a stream under the room with that streamId. 
Check the screenshots below. I've added streamId parameter as test1234
<img width="1593" alt="Screenshot 2023-06-03 at 16 25 51" src="https://github.com/ant-media/Ant-Media-Server/assets/3456251/b307054e-a90d-425a-839c-10d3f7a1da4c">

I see that it's published as separate

<img width="1717" alt="Screenshot 2023-06-03 at 16 26 09" src="https://github.com/ant-media/Ant-Media-Server/assets/3456251/f74e990a-b7f0-4690-bf1d-ff465c2798d2">

If you mean the specifying the room Id, I think we can add this option to the multitrack-conference.html or you can add it and make a PR :) 

Have a great weekend ;)

Regards
Oguz

**Reply by AppThisThing** [reply]
**Awesome**, my misunderstanding.

Once again, thanks for the quick response and support!

 Thank you.
You also - enjoy

**Reply by mekya** [reply]
My pleasure :)

Cheers
